pkg/bindinfo/norm/normalize.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package norm
+
+import (
+	"strings"
+
+	"github.com/pingcap/tidb/pkg/parser"
+	"github.com/pingcap/tidb/pkg/parser/ast"
+	utilparser "github.com/pingcap/tidb/pkg/util/parser"
+)
+
+type option struct {
+	specifiedDB string
+	fuzz        bool
+}
+
+type optionFunc func(*option)
+
+// WithFuzz specifies whether to eliminate schema names.
```

Comment:
I support separate the normalize function to a single .go file. But do we need to create a new subpackage and put this .go file into subpackage ?

Reply:
```normalize``` will not frequently update in the future. so this package's cache will not always be changed. It can make build/lint faster. Additionally, we can better divide responsibilities with ```bindinfo```.